src/Clock.tsx

```diff
@@ -0,0 +1,52 @@
+import { Component } from 'react';
+
+type Props = {
+  name: string
+};
+
+type State = {
+  time: string,
+};
+
+export class Clock extends Component<Props, State> {
+  state = {
+    time: new Date().toUTCString().slice(-12, -4),
+  };
+
+  intrval = 0;
+
+  componentDidMount() {
+    this.intrval = window.setInterval(() => {
+      this.setState({ time: new Date().toUTCString().slice(-12, -4) });
+      console.info(this.state.time);
```

Comment:
Let's disable eslint in this line to make the tests work
![image](https://user-images.githubusercontent.com/96409563/205588934-73071e92-c7e8-47af-b276-0b5c6d94e2bb.png)